selfdrive/ui/ui.h

```diff
@@ -78,19 +78,22 @@ struct Alert {
       // Handle controls timeout
       if (controls_frame < started_frame) {
         // car is started, but controlsState hasn't been seen at all
-        alert = {"openpilot Unavailable", "Waiting for controls to start",
+        alert = {QT_TRANSLATE_NOOP("OnroadAlerts", "openpilot Unavailable"),
```

Comment:
`Alert` seems fine to me, the parent class isn't OnroadAlerts anyway

Reply:
y, QT_TR_NOOP is better. there's also another issue with the previous code, which didn't call the qApp->translate for translation.